fvm/blueprints/scripts/systemChunkTransactionTemplateDualAuthorizer.cdc

```diff
@@ -0,0 +1,18 @@
+import FlowEpoch from 0xEPOCHADDRESS
+import NodeVersionBeacon from 0xNODEVERSIONBEACONADDRESS
+
```

Comment:
Probably would be good to add a comment explaining what this does at a high level.

Reply:
I think I'll add those in the code, not in the transaction itself, to save on some transaction bytes, since this one gets sent a lot.